docs/Training-ML-Agents.md

```diff
@@ -552,3 +556,29 @@ Some considerations:
 - **Result Variation Using Concurrent Unity Instances** - If you keep all the
   hyperparameters the same, but change `--num-envs=<n>`, the results and model
   would likely change.
+
+### Using PyTorch (Experimental)
+
+ML-Agents, by default, uses TensorFlow as its backend, but experimental support
+for PyTorch has been added. To use PyTorch, the `torch` Python package must
+be installed, and PyTorch must be enabled for your trainer.
+
+#### Installing PyTorch
+
+Installing PyTorch can usually be done by running `pip install torch`
```

Comment:
Should we also mention that you can do it when installing ml-agents with
```
pip install -e ./ml-agents[torch]
```

Reply:
Actually, shouldn't `pip3 install mlagents[torch]` work as well?

Reply:
I updated with `pip3 install mlagents[torch]` and a link to the installation instructions.